### PR TITLE
Harvesting-status

### PR DESCRIPTION
**Pull request by segfaultxavi**

### CODE REVIEW

**Comment by zachamo** [comment]
I like this -- it's a bit verbose, but there is value in having tidbits like this in the end user guides so that folks like node operators and external project teams have a bit of a bread crumb trail for troubleshooting.. Clarifying the FAILED status like this could save a lot of headache for someone thinking a node is broken when it is in fact working. 

Edits I would consider:
- Reorder the bullets in the FAILED Status box so that the Statistics Service is 2nd in the list -- if a node isn't listed here, it will always show as FAILED because the wallet cannot see the node's URL.. That said, this would be a short-term edit until the Statistics Service consistently shows all nodes.. I do wonder how this impacts the testnet though, as I couldn't properly sync using catapult 1.1.1 and had to move to 1.1.2alpha --> leaving it as the 2nd entry might be helpful for testers struggling with this (assuming there is a stats service for the testnet)
- on the FAILED status table entry: "Activation did not succeed or could not be verified. **It is recommended to select a different node**[...]" - consider "**Selecting a different node is the recommended solution**" -- "recomended to" just feels awkward. 
- "This status can be due to a number of reasons and there is not much to do to fix it besides changing to a different node or wait a bit." --> "Several different scenarios can c- on the FAILED status..  "Activation did not succeed or could not be verified. **It is recommended to select a different node**[...]" - consider "**Selecting a different node is the recommended solution**" -- "recomended to" just feels awkward. 
- "This status can be due to a number of reasons and there is not much to do to fix it besides changing to a different node or wait a bit." --> "Several different scenarios can cause this status and there is no easy way for a harvester to know the cause. The best solution for most harvesters it to select a different node, though some harvesters may prefer to contact the node operator directly or wait for a resolution ."ause this status and there is no easy way for a harvester to know the cause. The best solution for most harvesters it to select a different node, though some harvesters may prefer to contact the node operator directly or wait for a resolution ."
- You COULD link to https://symbol.github.io/symbol-openapi/v1.0.3/#operation/getUnlockedAccount in relation to "The node could not be contacted to verify the activation", but that's getting far too deep for most users, and is further complicated by the private key response rather than the user-facing address... This actually makes me think there's value in a new entry for _Guides > Network > Remote API node checks with a Browser_  which would outline some basic API calls you can pop into your browser for height, unlocked accounts, health, etc.. I could draft something after the holidays.

Sorry for nitpicking!

**Comment by segfaultxavi** [comment]
> * Reorder the bullets in the FAILED Status box so that the Statistics Service is 2nd in the list -- if a node isn't listed here, it will always show as FAILED because the wallet cannot see the node's URL.. That said, this would be a short-term edit until the Statistics Service consistently shows all nodes.. I do wonder how this impacts the testnet though, as I couldn't properly sync using catapult 1.1.1 and had to move to 1.1.2alpha --> leaving it as the 2nd entry might be helpful for testers struggling with this (assuming there is a stats service for the testnet)

Agreed. Done.

> * on the FAILED status table entry: "Activation did not succeed or could not be verified. **It is recommended to select a different node**[...]" - consider "**Selecting a different node is the recommended solution**" -- "recomended to" just feels awkward.

Yeah, agreed. Reworded.

> * "This status can be due to a number of reasons and there is not much to do to fix it besides changing to a different node or wait a bit." --> "Several different scenarios can cause this status and there is no easy way for a harvester to know the cause. The best solution for most harvesters it to select a different node, though some harvesters may prefer to contact the node operator directly or wait for a resolution ."

A bit too wordy for my taste, it's hard to keep the reader's attention for so long! 😀 
But I get your point and I tried to include it.

> * You COULD link to https://symbol.github.io/symbol-openapi/v1.0.3/#operation/getUnlockedAccount in relation to "The node could not be contacted to verify the activation", but that's getting far too deep for most users, and is further complicated by the private key response rather than the user-facing address... 

This is a guide for Wallet users, yeah, not your average REST API user! The manual version of the guide is mentioned, though, and that includes the link to the `unlockedaccounts` endpoint, so any tech savvy user has a thread to follow if interested.

> This actually makes me think there's value in a new entry for _Guides > Network > Remote API node checks with a Browser_  which would outline some basic API calls you can pop into your browser for height, unlocked accounts, health, etc.. I could draft something after the holidays.

It will be very welcome!

> Sorry for nitpicking!

THANKS for nitpicking!
